packages/gestalt/src/Video/Playhead.js

```diff
@@ -30,9 +30,14 @@ export default class VideoPlayhead extends PureComponent<Props, State> {
   seek: (clientX: number) => void = (clientX) => {
     if (this.playhead) {
       const { duration, seek } = this.props;
-      const { left, width } = this.playhead.getBoundingClientRect();
-      const percent = Math.max(0, Math.min((clientX - left) / width, 1));
+      const { left, right, width } = this.playhead.getBoundingClientRect();
+
+      const isRTL = window.getComputedStyle(this.playhead).direction === 'rtl';
```

Comment:
Good question. `Math.abs` because i wanted to aviod extra conditional statement. In RTL we have to calculate`right - clientX`, in contrast to LTR where we have to do `clientX - left`. Because if we do `clientX - right`, it gives us a negative value. So using `Math.abs` we can preserve the position of `clientX` in the formula.

Reply:
Alternatively, we could do this:

```
const difference = isRTL ? clientX - left : right - clientX;
```
@AlbertCarreras what do you think?

Reply:
I like the second one, just because it's a little clearer to read/debug without thinking about it